includes/class-wp-job-manager-shortcodes.php

```diff
@@ -69,7 +69,16 @@ public function __construct() {
 	public function shortcode_action_handler() {
 		global $post;
 
-		if ( is_page() && has_shortcode( $post->post_content, 'job_dashboard' ) ) {
+		/**
+		 * Determine if the shortcode action handler should run.
+		 *
+		 * @param bool $should_run_handler Should the handler run.
+		 *
+		 * @return bool
```

Comment:
As per the [guide](https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#4-hooks-actions-and-filters), the `@return` statement should not be used.